core/src/main/java/me/bogerchan/niervisualizer/util/KeyFrameMaker.kt

```diff
@@ -59,7 +51,9 @@ class KeyFrameMaker {
         val fftFrac = mFftAnimator.computeCurrentValue()
         if (mWaveAnimator.hasValueUpdated) {
             computedWaveData.originMap { idx, _ ->
-                (((mDestWaveData[idx].toInt() and 0xFF) - (mPrevWaveData[idx].toInt() and 0xFF)) * waveFrac + (mPrevWaveData[idx].toInt() and 0xFF)).toByte()
+                (((mDestWaveData[idx].toInt() and 0xFF) -
+                    (mPrevWaveData[idx].toInt() and 0xFF)) * waveFrac + (mPrevWaveData[idx].toInt() and 0xFF)
+                ).toByte()
```

Comment:
If the code is same as previously, please avoid adding formatting changes @sergeykrupenich

Reply:
removed